### PR TITLE
fix #74653 downfile.site

### LINKED ISSUE #74653: downfile.site
### Issue URL (Anti Adblock Script)
 [https://downfile.site/huong-dan-cai-dat-he-thong-monitor-voi-grafana-influxdb-va-telegraf-tren-centos-7-2.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fdownfile.site%2Fhuong-dan-cai-dat-he-thong-monitor-voi-grafana-influxdb-va-telegraf-tren-centos-7-2.html)
### Comment
Username: `@BarhamBasha`
> Source: http://megafly.in/JzvWM

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Firefox
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> BarbBlock<br/><br/><b>Language-specific:</b><br/> Liste AR

### PR DESCRIPTION
**Pull request by wout-codes**
#74653 
Added `cryptojunkie.net` from my discovery.

### CODE REVIEW

**Comment by wout-codes** [comment]
@Yuki2718 Any ideas on what to add to the `content_blocker.txt`?

**Comment by wout-codes** [comment]
@zloyden What should I add for `content_blocker.txt`?

**Comment by Yuki2718** [comment]
@wout-codes `@@||downfile.site^$generichide`

**Comment by wout-codes** [comment]
Thanks

**Comment by Yuki2718** [comment]
Sorry I was wrong, there's a special section to disable 
```
.pub_300x250
.pub_300x250m
.pub_728x90
.text-ad
.textAd
.text_ad
.text_ads
.text-ads
.text-ad-links
```

**Comment by wout-codes** [comment]
> Sorry I was wrong, there's a special section to disable
> 
> ```
> .pub_300x250
> .pub_300x250m
> .pub_728x90
> .text-ad
> .textAd
> .text_ad
> .text_ads
> .text-ads
> .text-ad-links
> ```
> 
> [EDIT] Actually there _are_. Better to wait for @zloyden ' reply.

@Yuki2718 This site `downfile.site` has already been added to those rules before but it doesn't work anymore. Requires update so the scriptlet work in this case.

**Comment by Yuki2718** [comment]
@wout-codes The existing rule is a css rule to force-display them and works perfectly if you disable bab and fab scriptlet. CB doesn't support css rule nor scriptlet.

**Comment by wout-codes** [comment]
Okay in that case I will delete the domain from Fab script, will it work?

**Comment by Yuki2718** [comment]
No need to delete when fab script is still there, CB anyway ignores scriptlet and for other platforms the scriptlet you added should work.

File: EnglishFilter/sections/content_blocker.txt [hunk 1]
```diff
@@ -1560,6 +1561,8 @@ glav.su#@#.c-bl-message
 @@||haaretz.co.il^$elemhide
 ! https://github.com/AdguardTeam/AdguardFilters/issues/57530
 @@||whatsaero.com^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/74653
+@@||downfile.site^$generichide
```
Comment (zloyden) [inline EnglishFilter/sections/content_blocker.txt:1565]: It's ok for content blockers. There are limited options for those platforms.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/content_blocker.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/content_blocker.txt:1565]: Ok.

**Comment by Yuki2718** [comment]
Well, but should be placed on top - above `! https://github.com/AdguardTeam/AdguardFilters/issues/74336` line. I'll move the rule.

**Pull request merged**